src/main/java/TaskList.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+public class TaskList {
+    ArrayList<Task> tasklist = new ArrayList<Task>();
+
+    public void listTasks() {
+        int i = 1;
+        for (Task task:tasklist) {
+            if (task.completed) {
+                System.out.println(" " + i + ".[X] " + task.taskName);
+            } else {
+                System.out.println(" " + i + ".[ ] " + task.taskName);
+            }
+            i++;
```

Comment:
A for loop will be much cleaner

Reply:
I think it is probably okay, it's another format of for loop